lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/ContextManager.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
In fact, that was the reason I was simply put a List in driver class, expecting more changes in future. We cannot put all the changes in one PR but focus what we want mainly change in the JIRA.